module/mempool/stdmap/backdata/mapBackData.go

```diff
@@ -82,6 +82,26 @@ func (b MapBackData) All() map[flow.Identifier]flow.Entity {
 	return entities
 }
 
+func (b MapBackData) Identifiers() flow.IdentifierList {
+	ids := make(flow.IdentifierList, len(b.entities))
+	i := 0
+	for entityID := range b.entities {
+		ids[i] = entityID
+		i++
+	}
+	return ids
+}
+
+func (b MapBackData) Entities() []flow.Entity {
+	entities := make([]flow.Entity, len(b.entities))
+	i := 0
+	for _, entity := range b.entities {
```

Comment:
maybe replace 'range' with 'for' loop

Reply:
`b.entities` is a map and we may not be able to iterate on it using a for loop unless through reflection. Not sure if it is the desired case?

Reply:
ok